install/configure.sh

```diff
@@ -6,6 +6,32 @@
 
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+echo ROOT=${ROOT}
+
+if [ ! -d "${ROOT}/.clean_" ]
+then
+    echo "Initializing CWS directory... "
+    mkdir ${ROOT}/.clean_
+    # Save clean CWS before configure.sh is run and directory is modified
+    rsync -av --exclude='bpmn' ${ROOT}/* ${ROOT}/.clean_ > /dev/null 2>&1
+else
+    echo "Re-running CWS Config, backing up current configuration. Refer to .backups folder for all backups."
+
+    # Make .backups folder for current CWS property version
+    BACKUP_DIR=${ROOT}/.backups/backup_$(date '+%Y%m%d_%H%M%S')
+
+    mkdir -p ${BACKUP_DIR}
+    cp -r ${ROOT}/* ${BACKUP_DIR}
```

Comment:
This line relies on the shell's globbing settings to ignore dotfiles by default (i.e. the `.backups/` folder), which is not portable and potentially unsafe. 

Prefer running in a subshell where we can enforce dotglob unset (see [here](https://unix.stackexchange.com/a/339583)):
```
(shopt -u dotglob; cp -R ${ROOT}/* ${BACKUP_DIR}/)
```

(Also switches to `-R`, as `-r` [is not portable](https://pubs.opengroup.org/onlinepubs/009695399/utilities/cp.html).)